features/fields-fout-cases.feature

```diff
@@ -0,0 +1,193 @@
+    | instance | /haalcentraal/api/brp/personen                                                                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code   | name   | reason                                           |
+    | fields | fields | Deel van de parameterwaarde niet correct: <pad>. |
+
+    Voorbeelden:
+    | pad                                  |
+    | voorletters                          |
+    | voornamen                            |
+    | adellijkeTitelPredicaat              |
+    | adellijkeTitelPredicaat.code         |
+    | adellijkeTitelPredicaat.omschrijving |
+    | adellijkeTitelPredicaat.soort        |
+    | voorvoegsel                          |
+    | geslachtsnaam                        |
+    | datum                                |
+    | plaats                               |
+    | land                                 |
+    | type                                 |
+    | code                                 |
+    | plaats.code                          |
+    | land.code                            |
+    | omschrijving                         |
+    | plaats.omschrijving                  |
+    | land.omschrijving                    |
+    | soort                                |
+    | datumIngangGeldigheid                |
+    | indicatieOnbekend                    |  
+
+  @fout-case
+  Scenario: Fields parameter bevat spaties
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999994086                       |
+    | fields              | burgerservicenummer, naam       |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                                                                      |
+    | type     | https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?#System_Net_HttpStatusCode_BadRequest |
+    | title    | Een of meerdere parameters zijn niet correct.                                                               |
+    | status   | 400                                                                                                         |
+    | detail   | De foutieve parameter(s) zijn: fields.                                                                      |
+    | code     | paramsValidation                                                                                            |
+    | instance | /haalcentraal/api/brp/personen                                                                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code    | name   | reason                                              |
+    | pattern | fields | Waarde voldoet niet aan patroon ^[a-zA-Z0-9\.,_]+$. |
+
+  @fout-case

```

Comment:
Ik heb nog even gekeken of we een dubbele komma met een regular expression kunnen afvangen en dat kan.
We hebben nu het pattern '^[a-zA-Z0-9\.,_]+$', als we dat vervangen door '^(((?!,,))[a-zA-Z0-9\.,_])+' dan worden dubbele komma's niet meer geaccepteerd.
Om ook dubbele punten te kunnen weigeren gebruik je '^(((?!,,|\.\.))[a-zA-Z0-9\.,_])+'.

Reply:
@melsk-r prima suggestie, maar niet onderdeel van deze pull request. zou je hiervoor een issue kunnen maken?